docs/content/technical-specifications.md

```diff
@@ -22,7 +22,7 @@ We explicitly test the NGINX Ingress Controller (NIC) on a range of Kubernetes p
 {{% table %}}
 | NIC Version | Supported Kubernetes Version | NIC Helm Chart Version | NIC Operator Version | NGINX / NGINX Plus version |
 | --- | --- | --- | --- | --- |
-| 2.3.1 | 1.24 - 1.19 | 0.14.1 | 1.1.0 | 1.23.1 / R27 |
+| 2.4.0 | 1.24 - 1.19 | 0.14.1 | 1.1.0 | 1.23.1 / R27 |
```

Comment:
We need a new line here instead of a changed line, and the values need to be updated
```suggestion
| 2.4.0 | 1.25 - 1.19 | 0.15.0 | 1.2.0 | 1.23.1 / R27 |
| 2.3.1 | 1.24 - 1.19 | 0.14.1 | 1.1.0 | 1.23.1 / R27 |
```